### ISSUE TITLE
Any chance of new development for the library?

### ISSUE THREAD

**Issue by volumia** [bakpakin/Splash.lua#3]
I've been looking for a collision library with various collision primitives for a while now to escape Box2D, but Hardon Collider doesn't provide enough collision resolution info (like the hit normal), and bump.lua is AABB only, so Splash seems like a great fit for me.  
However, it seems all progress has been ceased. I'd normally have no issue with this; I can fork it and resume progress on my own (I have), but there isn't enough code documentation for me to do anything very substantial.  
Lots of sweep tests haven't been implemented (like AABB vs Segment, which I want for slopes), and Segment vs Segment, among other things.  
Is there any chance of it resuming progress in the future, or at least code documentation? Thanks!

**Comment by bakpakin** [comment]
I haven't worked on this for a while, simply because I have moved on to other things. I would be willing to revisit and help add features if you would like.

The largest problem I ran into was resolving circle to circle collisions. The detection was simple, but resolution proved quite difficult for some reason, especially in situations with more than 3 circles.

The math for the sweep tests got a bit annoying , but that is doable.
